README.md

```diff
@@ -37,7 +39,7 @@ Ruby | 🚧
 
 <!-- toc -->
 * [Supported version control systems](#Which-version-control-system-can-you-use)
-* [Getting Started](#Getting-Started)
+* [Getting Started](#Getting-Started-🏁)
```

Comment:
It doesn't work. See https://github.com/DXHeroes/dx-scanner/tree/docs/add-slack-invite